[Flake] in windows/quickstart-cmake-dll-presubmit

The first CI run of https://github.com/googleapis/google-cloud-cpp/pull/3802, I saw an unrelated breakage from the `windows/quickstart-cmake-dll-presubmit` build. I triggered a re-run and it passed:

https://source.cloud.google.com/results/invocations/cb79e1c3-57cd-4420-94bd-467a3971c1e1/targets/cloud-devrel%2Fclient-libraries%2Fcpp%2Fgoogle-cloud-cpp%2Fwindows%2Fquickstart-cmake-dll-presubmit/log

A snippet of the error:

```
20-04-16T14:25:13.9485942+00:00 Compiling with CMake Debug
[1/2] Building CXX object CMakeFiles\quickstart.dir\quickstart.cc.obj
T:\src\github\google-cloud-cpp\cmake-out\vcpkg-quickstart\installed\x64-windows\include\google/protobuf/stubs/platform_macros.h(134): warning C4005: 'PROTOBUF_USE_DLLS': macro redefinition
T:\src\github\google-cloud-cpp\google\cloud\bigtable\quickstart\quickstart.cc: note: see previous definition of 'PROTOBUF_USE_DLLS'
T:\src\github\google-cloud-cpp\cmake-out\vcpkg-quickstart\installed\x64-windows\include\google/protobuf/stubs/logging.h(102): warning C4251: 'google::protobuf::internal::LogMessage::message_': class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' needs to have dll-interface to be used by clients of class 'google::protobuf::internal::LogMessage'
c:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.21.27702\include\xstring(4231): note: see declaration of 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>'
T:\src\github\google-cloud-cpp\cmake-out\vcpkg-quickstart\installed\x64-windows\include\google/protobuf/io/coded_stream.h(1250): warning C4251: 'google::protobuf::io::CodedOutputStream::default_serialization_deterministic_': struct 'std::atomic<bool>' needs to have dll-interface to be used by clients of class 'google::protobuf::io::CodedOutputStream'
c:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.21.27702\include\atomic(1817): note: see declaration of 'std::atomic<bool>'
T:\src\github\google-cloud-cpp\cmake-out\vcpkg-quickstart\installed\x64-windows\include\google/protobuf/arena_impl.h(287): warning C4251: 'google::protobuf::internal::ArenaImpl::lifecycle_id_generator_': struct 'std::atomic<__int64>' needs to have dll-interface to be used by clients of class 'google::protobuf::internal::ArenaImpl'
```

--- a/ci/kokoro/windows/build.ps1
+++ b/ci/kokoro/windows/build.ps1
@@ -122,7 +122,7 @@ if (Test-Path env:KOKORO_ARTIFACTS_DIR) {
             -Exclude test.xml,sponge_log.xml,build.bat `
             -ErrorAction SilentlyContinue `
             -Path "${env:KOKORO_ARTIFACTS_DIR}" | `
-            Remove-Item -Recurse -Force -ErrorAction SilentlyContinue    
+            Remove-Item -Recurse -Force -ErrorAction SilentlyContinue
     } catch {
         Write-Host "$(Get-Date -Format o) error cleaning up KOKORO_ARTIFACTS DIR, ignored"
     }

--- a/ci/kokoro/windows/build.ps1
+++ b/ci/kokoro/windows/build.ps1
@@ -116,13 +116,18 @@ $BuildExitCode = $LastExitCode
 # Remove most things from the artifacts directory. Kokoro copies these files
 # *very* slowly on Windows, and then ignores most of them :shrug:
 if (Test-Path env:KOKORO_ARTIFACTS_DIR) {
-    Set-Location "$env:KOKORO_ARTIFACTS_DIR"
-    Get-ChildItem -Recurse -File `
-        -Exclude test.xml,sponge_log.xml,build.bat `
-        -ErrorAction SilentlyContinue | `
-        Remove-Item -Recurse -Force -ErrorAction SilentlyContinue
+    try {
+        $ErrorActionPreference = "SilentlyContinue"
+        Get-ChildItem -Recurse -File `
+            -Exclude test.xml,sponge_log.xml,build.bat `
+            -ErrorAction SilentlyContinue `
+            -Path "${env:KOKORO_ARTIFACTS_DIR}" | `
+            Remove-Item -Recurse -Force -ErrorAction SilentlyContinue    
+    } catch {
+        Write-Host "$(Get-Date -Format o) error cleaning up KOKORO_ARTIFACTS DIR, ignored"
+    }
 }
 
 if ($BuildExitCode) {
-    throw "Build failed with exit code $LastExitCode"
+    throw "Build failed with exit code ${BuildExitCode}"
 }

